api/users/serializers.py

```diff
@@ -142,6 +180,10 @@ def update(self, instance, validated_data):
                                 detail='{} only accept a list of one single value'. format(key)
                             )
                         instance.social[key] = val[0]
+            elif 'schools' == attr:
+                instance.schools = value
```

Comment:
I don't think these assignment statements are necessary anymore. The fields will be set by the `setattr(instance, attr, value)` call below.